web/config.toml

```diff
@@ -29,6 +29,12 @@ staticDir = ["dist", "static"]
   identifier = "docs"
   weight = 4
 
+[[menu.main]]
+  name = "OGS-5"
```

Comment:
Maybe in the top bar: 

 - OpenGeoSys__-6__ 
 - OpenGeoSys-5

for consistent naming.